gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.9-bin.zip
```

Comment:
We need a explicit wrapper version on the build script

Reply:
It's not technically necessary, but it's probably clearer to add it.

Reply:
I prefer so, because it is easier to read the requires only with the gradle build script.

Reply:
Version 4.10 is already here - does it break something that we need?

Reply:
Oh, no, I just didn't realize 4.10 was out.  Will update.